src/Illuminate/Database/Migrations/stubs/create.stub

```diff
@@ -13,7 +13,7 @@ class DummyClass extends Migration
      */
     public function up()
     {
-        Schema::create('DummyTable', function (Blueprint $table) {
+        Schema::createIfNotExists('DummyTable', function (Blueprint $table) {
```

Comment:
This PR looks great, but I don't think we should change the default. Silent failures are not great.

Reply:
@GrahamCampbell reverted to original in new commit.